extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/supervisor/KafkaSupervisorTest.java

```diff
@@ -2532,6 +2549,9 @@ public void testFailedInitializationAndRecovery() throws Exception
 
   private void addSomeEvents(int numEventsPerPartition) throws Exception
   {
+    //create topic manually
+    AdminUtils.createTopic(zkUtils, topic, NUM_PARTITIONS, 1, new Properties(), RackAwareMode.Enforced$.MODULE$);
```

Comment:
Hmm, would you tell me why this is needed?

Reply:
Haha, it looks like there was something wrong in Github. Got it. Thanks.

Reply:
Sorry about that ~ I will try to delete the duplicate comment~